src/bulk/common.ts

```diff
@@ -26,7 +26,10 @@ import type { Collection } from '../collection';
 import type { Topology } from '../sdam/topology';
 import type { CommandOperationOptions, CollationOptions } from '../operations/command';
 import type { Hint } from '../operations/operation';
-import type { Filter, OptionalId, UpdateFilter } from '../mongo_types';
+import type { Filter, OneOrMore, OptionalId, UpdateFilter } from '../mongo_types';
+
+/** @internal */
+const kErr = Symbol('err');
```

Comment:
```suggestion
const kServerError = Symbol('serverError');
```
Can we name this a bit more descriptively?